.gitlab-ci.yml

```diff
@@ -6,6 +6,7 @@ stages:
   - 100_cell_test
 
 before_script:
+  - export CI_COMMIT_REF_NAME=integration
```

Comment:
This line, and the rest of the changes to this file, should probably be removed.

Reply:
Take a look at the commits. The second is clearly marked to indicate this.

Reply:
Ok. Still confused about it's presence in the PR.

Reply:
What exactly are you confused about? This is a temporary commit that is needed for the PR to run and pass on Gitlab. The commit will be removed before the PR is merged.

Reply:
I was surprised that the PR contained changes not intended for review, and that this should have been obvious by examination of the commit history. Thank you for clarifying.